Firestore/core/src/firebase/firestore/model/field_path.h

```diff
@@ -47,10 +48,28 @@ class FieldPath : public impl::BasePath<FieldPath> {
   FieldPath(const IterT begin, const IterT end) : BasePath{begin, end} {
   }
   FieldPath(std::initializer_list<std::string> list) : BasePath{list} {
+    if (size() == 0) {
```

Comment:
Nit: `if (empty())` would, IMO, be a little more idiomatic.

Reply:
good point, done!